src/vivarium_nih_us_cvd/components/healthcare_utilization.py

```diff
@@ -215,34 +216,19 @@ def on_time_step_cleanup(self, event: Event) -> None:
             visit_background, data_values.COLUMNS.VISIT_TYPE
         ] = data_values.VISIT_TYPE.BACKGROUND
 
-        # Take measurements (required to determine if a followup is required)
+        # Schedule followups
         all_visitors = visit_emergency.union(visit_scheduled).union(visit_background)
-        measured_sbp = self.treatment.get_measured_sbp(
-            index=all_visitors,
-            mean=data_values.MEASUREMENT_ERROR_MEAN_SBP,
-            sd=data_values.MEASUREMENT_ERROR_SD_SBP,
+        to_schedule_followup_sbp = self.determine_followups_sbp(
```

Comment:
Note that the original lines 218-242 are just copy/pasted into the method self.determine_followups_sbp().

Feel free to assume that's correct and just focus on the new logic (self.determine_followups_ldlc())